### PR TITLE
S2U-45 5.2.6.2 Roster: Add a column for each Section & Group to the Export

### PR DESCRIPTION
**Pull request by V100X**
Jira: [S2U-45](https://sakaiproject.atlassian.net/browse/S2U-45)

Result example:

sectionsSheet

[S2U-45]: https://sakaiproject.atlassian.net/browse/S2U-45?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: roster2/bundle/src/bundle/roster.properties [hunk 1]
```diff
@@ -75,6 +75,8 @@ facet_status = Status
 facet_credits = Credits
 facet_groups = Groups
 facet_roster = Roster
+facet_sections = Sections
```
Comment (mpellicer) [inline roster2/bundle/src/bundle/roster.properties:78]: Put them at the end of the file with a comment "# S2U-45"

File: roster2/bundle/src/bundle/roster_ca.properties [hunk 2]
```diff
@@ -75,6 +75,8 @@ facet_status=Estat
 facet_credits=Cr\u00e8dits
 facet_groups=Grups
 facet_roster=Orla
+facet_sections=Seccions
```
Comment (mpellicer) [inline roster2/bundle/src/bundle/roster_ca.properties:78]: Put them at the end of the file with a comment "# S2U-45"

File: roster2/bundle/src/bundle/roster_es.properties [hunk 3]
```diff
@@ -75,6 +75,8 @@ facet_status=Estado
 facet_credits=Cr\u00e9ditos
 facet_groups=Grupos
 facet_roster=Participantes
+facet_sections=Secciones
```
Comment (mpellicer) [inline roster2/bundle/src/bundle/roster_es.properties:78]: Put them at the end of the file with a comment "# S2U-45"

File: roster2/bundle/src/bundle/roster_eu.properties [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+screenreader_comboBox=Pulse Alt + cursor (arriba o abajo) para desplazarse por el men\u00fa
+name_pronunciation_not_provided=[No proporcionado]
+name_pronunciation_profile_link=Introduzca la pronunciaci\u00f3n de su nombre en el Perfil
+roster_user_groups_label=Selector donde se muestran los grupos
+
+# S2U-45
+facet_sections=Secciones
+facet_sections_category=Secciones de la categor\u00eda:\u0020
+
+# THIS FILE WILL BE TRANSLATED by: Pablo San Roman
```
Comment (mpellicer) [inline roster2/bundle/src/bundle/roster_eu.properties:115]: @pablosanroman I feel this file should be translated, but you can also provide the last two values.

**Review by mpellicer — APPROVED** [review]
Looks very nice good job @V100X

**Comment by V100X** [comment]
Thanks! 🙌

**Review by jesusmmp — APPROVED** [review]
LGTM!

**Review by mpellicer — APPROVED** [review]

**Pull request merged**